solvers/infomax.py

```diff
@@ -0,0 +1,32 @@
+from benchopt import BaseSolver, safe_import_context
+
+
+with safe_import_context() as import_ctx:
```

Comment:
Note that with the latest release, you should not require the `safe_import_context` anymore